app/src/main/java/com/androiddev/social/timeline/ui/ReplyBox.kt

```diff
@@ -315,15 +477,56 @@ fun PhotoPickerResultComposable(addUri: (Uri) -> Unit, clearFocus: () -> Unit) {
 
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun PollCreator(
+    focusRequester: FocusRequester,
+    pollOptionAdded: (String) -> Unit,
+) {
+    var option: TextFieldValue by remember { mutableStateOf(TextFieldValue("")) }
+
+    TextField(
+        value = option,
+        onValueChange = {
+            option = it
+        },
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(
+                horizontal = PaddingSize1,
+                vertical = PaddingSize2
+            )
+            .focusRequester(focusRequester),
+        label = {
+            Text("Type an option for the poll, then make it done from keyboard.")
```

Comment:
Make it done? Sounds a bit off

Reply:
:)) yep! I'm not a UI/UX designer! In this sentence, I want to say that user should use the keyboard's done button to add the option of the poll. Maybe It needs a button, instead of this hint. I'll fix it.

Reply:
Sorry for the long delay! I didn't have time. But here we go. Let me know if it's okay. [6e19c60](https://github.com/digitalbuddha/Firefly/pull/16/commits/6e19c60dc92d647424128ddbf3021ec82ecf8c24)